src/services/helpers/gantt/color/confidenceColor.ts

```diff
@@ -0,0 +1,58 @@
+import { colors } from "../../../../theme/colors";
+
+// typescript enforces that the key has to match exactly whats in the dictionary.
+export function getConfidenceColor(
```

Comment:
It looks like this was just moved here, but added in a separate PR, we may wanna do some refactoring on this in the future.

Reply:
agreed